drivers/SmartThings/philips-hue/profiles/plug.yml

```diff
@@ -0,0 +1,12 @@
+name: SmartPlug
```

Comment:
It's not required, but I think we typically try to make the `name` match the filename

Reply:
Fixed on https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1359